Assets/Scripts/EndTaskInteraction.cs

```diff
@@ -95,6 +98,18 @@ void FixedUpdate()
             endTestTrigger.SetActive(true);
             mistakeTriggerParent.SetActive(true);
             shortcutTriggerParent.SetActive(true);
+
+            if (!angryBeeSpawned)
+            {
+                //Spawn bee at position and set player for targetting to be the player object
+                GameObject bee = Instantiate(angryBee, beeSpawnPoint.transform.position, Quaternion.identity);
+                bee.GetComponent<BeeAngryController>().player = gameObject.transform.parent.gameObject.transform;
+
+                //apply the bee object to the endtest script in the parent object, so we can remove when we reach the cave
+                gameObject.transform.parent.gameObject.GetComponent<EndTest>().angryBee = bee;
+
+                angryBeeSpawned = true;
```

Comment:
Just better code practice
```suggestion
                hasAngryBeeSpawned = true;
```